components/KiboHamburgerMenu.vue

```diff
@@ -38,7 +38,11 @@
             <div class="category-content">
               <SfList>
                 <SfListItem v-for="(category, key) in megaMenuCategories" :key="key">
-                  <SfMenuItem :label="$t(category.content.name)" @click="goNext(category)" />
+                  <SfMenuItem
+                    :label="$t(category.content.name)"
```

Comment:
remove $t, we won't have translations for this dynamic data here

Reply:
Resolved..